packages/apps/docs/src/pages/docs/contribute/index.tsx

```diff
@@ -1,38 +1,101 @@
-import { Stack } from '@kadena/react-ui';
+import { Box, Button, Card, Heading, Stack, Text } from '@kadena/react-ui';
 
 import { BrowseSection } from '@/components';
+import {
+  cardClass,
+  cardSectionClass,
+} from '@/components/Layout/Home/styles.css';
 import {
   checkSubTreeForActive,
   getPathName,
 } from '@/utils/staticGeneration/checkSubTreeForActive.mjs';
 import type { GetStaticProps } from 'next';
+import Link from 'next/link';
 import type { FC } from 'react';
 import React from 'react';
 
 const Home: FC = () => {
   return (
-    <Stack direction="column" gap="$2xl">
-      <BrowseSection>
-        <BrowseSection.LinkBlock
-          title="Run a Node"
-          subtitle=" "
-          icon="RestApi"
-          href="node/overview"
-        />
-        <BrowseSection.LinkBlock
-          title="Ambassadors"
-          subtitle=" "
-          icon="Contribute"
-          href="ambassadors/overview"
-        />
-        <BrowseSection.LinkBlock
-          title="Kadena DAO"
-          subtitle=" "
-          icon="Concepts"
-          href="kadena-dao"
-        />
-      </BrowseSection>
-    </Stack>
+    <>
+      <div className={cardSectionClass}>
+        <section className={cardClass}>
+          <Heading as="h4">Kadena DAO</Heading>
+          <Box marginY="$4">
+            <Text>
+              Kadena has built the framework for the first DAO, Decentralized
+              Autonomous Organization, on its public blockchain called dao.init.
+              As Kadena’s ecosystem continues to grow, the creation of a DAO
+              will allow for the broader community to provide their input in a
+              decentralized manner.
+            </Text>
+          </Box>
+          <Button as="a" asChild>
+            <Link href={'/docs/contribute/kadena-dao'}>Start contributing</Link>
+          </Button>
+        </section>
```

Comment:
imo this could be a separate docs component

Reply:
good point..maybe this was just lazyness from my side :)